lib/utilities/hardware.js

```diff
@@ -0,0 +1,27 @@
+import assert from 'bsert';
+import { HardwareWallet, tobip44Account } from './index';
```

Comment:
it definitely could be, but I thought that read kind of poorly

Reply:
Feels like it reads a little better to me, but I can see how it could go either way. Maybe we take the `bip` part out entirely? `derive44Account` or `deriveBIP44Account`? (was looking through bcoin, didn't find perfect correlation but did find methods like [this](https://github.com/bcoin-org/bcoin/blob/85ed59c842f2aa4c1a8154d74a9cad7696cdfee3/lib/wallet/wallet.js#L505))

Reply:
I feel like deriving implies cryptography, this is just templating a string